src/Http/Http/test/Internal/ItemsDictionaryTests.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.IO.Pipelines;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Primitives;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Http
+{
+    public class ItemsDictionaryTests
+    {
+        [Fact]
+        public void GetEnumerator_ShouldResolveWithoutNullReferenceException()
+        {
+            // Arrange
+            var dict = new Microsoft.AspNetCore.Http.ItemsDictionary();
```

Comment:
Add `using Microsoft.AspNetCore.Http;`

Reply:
This test class is already in that namespace, you shouldn't need the using or the prefix.

Reply:
Thanks!